tony-core/src/main/java/com/linkedin/tony/rpc/impl/TaskStatus.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.tony.rpc.impl;
+
+public enum TaskStatus {
+    NEW,
+    READY,
+    RUNNING,
+    FAILED,
+    SUCCEEDED;
+    TaskStatus() { }
```

Comment:
Is this constructor necessary? I think it's not required.

Reply:
This is just make it not public, i think it is an IDE warning

Reply:
hmm. it seems i don't have to put it here. removed